tox.ini

```diff
@@ -139,7 +141,7 @@ exclude=.md,
     packages/valory/protocols/ledger_api
     operate/data/
 max-line-length = 88
-select = B,C,D,E,F,I,W,
+select = B,C,D,E,F,I,W,PT
```

Comment:
PT = Pytest style. Suggestion to include it in [tomte repository](https://github.com/valory-xyz/tomte).